src/client/components/SentReviewsComponent/SentReviewsComponent.js

```diff
@@ -57,8 +57,26 @@ export const SentReviewsComponent = ({ reviews }) => {
   );
 };
 
+const Person = PropTypes.shape({
+  fullName: PropTypes.string.isRequired,
+  profileImageUrl: PropTypes.string,
+});
+
+const CV = PropTypes.shape({
+  fileUrl: PropTypes.string.isRequired,
+  averageStars: PropTypes.number.isRequired,
+  createdDate: PropTypes.instanceOf(Date).isRequired,
+  author: Person.isRequired,
+});
+
+const Review = PropTypes.shape({
+  id: PropTypes.number.isRequired,
+  review: PropTypes.string.isRequired,
+  createdDate: PropTypes.instanceOf(Date).isRequired,
+  reviewer: Person.isRequired,
+  cv: CV.isRequired,
+});
+
```

Comment:
Here you are kinda changing the structure of your PropTypes and how it relates to your data in storybook. By introducing new constants I think you will introduce new errors since my PropTypes followed your data structure. Did you have any PropType related errors in the console when running your code?

Reply:
I am not changing the structure, it is the same structure. Just introduced variables as I think makes it a bit easier to read.

Reply:
I know, but then I guess I will have to change my structure since I dont have those variables on my side. But its okay I guess

Reply:
And did you have any errors in your console about Proptypes?

Reply:
I don't think you should be getting an error from that. Before I didn't have the same propTypes as you. I was checking for an object while you were checking for a shape. You might just need the id. But you can try running the your code, at least in my case I don't have an error

Reply:
okay sounds good. if you dont have errors then thats a good start for a change on my side. I will have to do a bit more changes than that, to introduce the new variables I think but it is okay :) it is doable and yes, more readable I agree. 
Thank you :)

Reply:
Actually, there is an error from the stars component, but that one is because in that component we don't have keys for the elements of that array. It gives the same error that Ali pointed out from this component, so I will inform Hamed about that ;)

Reply:
Yes, i saw the stars were not rendering into the review.. i didn´t have time to investigate who´s story was that. Thank you for notifying him :)